js_modules/dagster-ui/packages/ui-core/src/asset-data/AssetHealthDataProvider.tsx

```diff
@@ -96,6 +96,10 @@ export const ASSETS_HEALTH_INFO_QUERY = gql`
       path
     }
 
+    assetMaterializations(limit: 1) {
+      timestamp
+    }
+
```

Comment:
@jamiedemaria I added this to the asset health query. Is this something that is slow to fetch?

Reply:
Ah can we do that then @jamiedemaria ? Or are we planning to move the asset health resolver onto GrapheneAsset at some point in order to properly handle external assets?

Reply:
I want to move the health resolver onto GrapheneAsset so that we can do external assets. Ideally i'll do that this week

Reply:
I don't want to block this though, so if it doesn't seem like i'll be able to start the move tomorrow, i'll just put up the Pr to add the sort key to grapheneassetnode and then i'll handle the move after